dwarf-fortress.sh

```diff
@@ -27,9 +27,11 @@ while getopts ":hsdR" opt; do
     case $opt in
         h)
             DFHACK="true"
+            TWBT="true"
             ;;
         d)
             DFHACK="false"
+            TWBT="false"
```

Comment:
Perhaps skip this, as noted above an unset variable will skip activating it. And since enabling TWBT requires a config change, there won't be a perfect way to disable it again through the script anyway.

Reply:
Done